code/modules/fluids/fluid_core.dm

```diff
@@ -787,7 +787,8 @@ ADMIN_INTERACT_PROCS(/obj/fluid, proc/admin_clear_fluid)
 	if (F.my_depth_level == 1)
 		if(!src.lying && src.shoes && src.shoes.hasProperty ("chemprot") && (src.shoes.getProperty("chemprot") >= 5)) //sandals do not help
 			do_reagent_reaction = 0
-			F.group.reagents.reaction(src.shoes, TOUCH, F.group.amt_per_tile, can_spawn_fluid = FALSE)
+			if (!src.wear_suit || !(src.wear_suit.body_parts_covered & LEGS)) // suits can go over shoes
```

Comment:
```suggestion
			if (!src.wear_suit || !(src.wear_suit.body_parts_covered & SPACEWEAR)) // suits can go over shoes
```
This makes more sense I think, since it's roughly what aqua tenebrae uses to determine protection.